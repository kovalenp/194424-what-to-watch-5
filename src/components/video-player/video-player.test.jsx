import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player";

describe(`VideoPlayer meets B21`, () => {

  it(`and rendered correctly for auth'ed`, () => {
    const tree = renderer.create((
      <VideoPlayer
        isPlaying={false}
        video={``}
        width={`1px`}
        height={`1px`}
      />
    )).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
