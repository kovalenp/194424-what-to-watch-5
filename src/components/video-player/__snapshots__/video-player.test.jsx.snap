// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer meets B21 and rendered correctly for auth'ed 1`] = `
<video
  controls={false}
  height="1px"
  muted={false}
  poster=""
  preload="none"
  src=""
  width="1px"
/>
`;
